index.js

```diff
@@ -53,9 +57,9 @@ function makePersonObject(/* code here */) {
  * For example, if we invoke `getName`
  * passing { id: 1, name: 'Leia', email: 'leia@leia.com` } as the argument,
  * the returned value should look like `Hello, my name is Leia`.
-*/
-function getName(/* code here */) {
-  /* code here */
+ */
+function getName({ id, name, email }) {
+    return `Hello, my name is ${name}`;
 }
```

Comment:
i can tell you have some experience.  I like the way you decomposed the object inside the function parameter.  You will find that to be useful in react.